app/controllers/names_controller.rb

```diff
@@ -133,11 +136,10 @@ def names_by_user
 
   # Display list of names that a given user is editor on.
   def names_by_editor
-    user = if params[:id]
-             find_or_goto_index(User, params[:by_editor].to_s)
-           else
-             @user
-           end
+    user = find_obj_or_goto_index(
+      model: User, obj_id: params[:by_editor].to_s,
+      index_path: names_path
+    )
```

Comment:
Changes functionality when by_user is not an existing user.
See [Comment about LocationsController](https://github.com/MushroomObserver/mushroom-observer/pull/1324/files#r1095161355).